docs/content/documentation/content/image-processing/index.md

```diff
@@ -145,3 +145,17 @@ Here is the result:
 <small>
   Image attribution: Public domain, except: _06-example.jpg_: Willi Heidelbach, _07-example.jpg_: Daniel Ullrich.
 </small>
+
+
+# Get image metadata
```

Comment:
The Tera fns are explained in ` docs/content/documentation/templates/overview.md` currently so this doc should be there. A link to it can still be added in this image processing doc.